code/datums/wounds/burns.dm

```diff
@@ -174,15 +185,15 @@
 			. += "\tSurgical debridement, antiobiotics/sterilizers, or regenerative mesh will rid infection. Paramedic UV penlights are also effective.\n"
 
 		if(flesh_damage > 0)
-			. += "Flesh damage detected: Please apply ointment or regenerative mesh to allow recovery.\n"
+			. += "Flesh damage detected: Application of ointment or regenerative mesh, or ingestion of \"Miner's Salve\", will speed flesh recovery. Good nutrition, rest, and keeping the wound clean can also slowly repair flesh.\n"
```

Comment:
Or application of synthflesh... Seems like Miner's Salve is being really specific?

Reply:
Well it's specific because it's one of the things that works